src/functionalTest/java/uk/gov/hmcts/reform/juddata/config/ParentCamelConfig.java

```diff
@@ -295,7 +296,7 @@ public DataLoadUtil dataLoadUtil() {
 
     @Bean
     IEmailService emailService() {
-        return mock(EmailServiceImpl.class);
+        return new EmailServiceImpl();
```

Comment:
its actual service now, will this trigger email for functional test cases ?

Reply:
The email is disabled in the functional tests configuration. So we won't receive any emails from them.